components/editor/primitives/PrimitiveContainer.tsx

```diff
@@ -1,14 +1,62 @@
 import { FC, ReactNode } from 'react'
+import clsx from 'clsx'
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/20/solid'
+
+const baseStyles: string =
+  'relative cursor-move space-y-2 border-2 border-dashed px-4 py-6 sm:rounded-lg sm:px-6'
 
 interface Props {
   children?: ReactNode
+  handleDownClick: Function
+  id: any
+  isOver: Boolean
+  showButtons: Boolean
+  handleUpClick: Function
 }
 
-const PrimitiveContainer: FC<Props> = ({ children }) => {
+const PrimitiveContainer: FC<Props> = ({
```

Comment:
Let's consolidate all the `Primitive` components into this one. Add a prop to pass in `options` with properties like name and type. These options will be fetched from the network.